src/serac/physics/materials/solid_functional_material.hpp

```diff
@@ -177,28 +177,6 @@ struct ConstantTraction {
   }
 };
 
-/// Function-based traction boundary condition model
```

Comment:
Was this deleted in favor of the custom integrals? I assume its due to GPU incompatibility.

Reply:
These were a source of confusion. To clarify: a q-function can be one of the following
- a lambda function (cpu-only) 
- a `std::function`(cpu-only) 
- a functor object (cpu and gpu)

The main reason to choose the functor approach is when you need GPU support, but the implementation of these functors doesn't work on the GPU. So, these classes were incurring unnecessary overhead, and making the requirements of q-function less clear.